src/SendEventLogEmail/Options.cs

```diff
@@ -22,6 +22,9 @@ public Options()
         private const string emailFromPasswordText = "emailfrompassword";
         private const string emailToText = "emailto";
         private const string emailSmtpText = "emailsmtp";
+        private const string ignoreInformationText = "ignoreinformation";
```

Comment:
Ah I see you are doing that down below , I was thinking more like in Options constructor. you can ignore the above message